scripts/push_docs_to_branch.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+TMP_DOCS_FOLDER=tmp-documentation
+TODAY=`date "+%Y%m%d"`
+
+# use a temp directory, clone the $DOCS_BRANCH in it
+echo "Cloning $GITHUB_REPOSITORY#$DOCS_BRANCH into $TMP_DOCS_FOLDER..."
+mkdir $TMP_DOCS_FOLDER
+git clone --depth=1 --branch $DOCS_BRANCH git@github.com:$GITHUB_REPOSITORY.git $TMP_DOCS_FOLDER
+
+echo "Updating the docs..."
+cp $DOCS_FOLDER/* $TMP_DOCS_FOLDER/
+cd $TMP_DOCS_FOLDER
+
+if [ -z "$NEW_VERSION" ]
+then
+    echo "Skipping generation of new version."
```

Comment:
in this case, is there actually anything we change? 🤔

Reply:
as in: I don't see anything we'd need to push in the lines after this `if`

Reply:
there might be changes under the `docs/` folder or the source files imported by the docs (the former `literalinclude` directive). That's why we copy them